File: Web/src/app/interfaces/sprint.ts
export interface sprint {
    companyId: number;
    projectId: number;
    sprintId: number;
    sprintName: string;
    description: string | null;
    creationDate: Date;
}


export interface sprintRequest {
    companyId: number;
    projectId: number;
    sprintId: number;
    sprintName: string;
    description: string | null;
    CreationUserId: number;
    Active: boolean;
}

File: Activity/Activity.Core/Entities/sprint.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Activity.Core.Entities
{
    public class sprint : baseEntity
    {
        public int CompanyId { get; set; }
        public int ProjectId { get; set; }
        public string SprintName { get; set; }
        public string Description { get; set; }
    }

    public class sprintRequest : sprint
    {
        public int SprintId { get; set; }
        public int CreationUserId { get; set; }
        public bool Active { get; set; }
    }
    public class sprintResponse : sprint
    {
        public DateTime CreationDate { get; set; }
        public List<projectUserResponse> ListProjectUser { get; set; } = new List<projectUserResponse>();
    }


}
